app/Validators/vm_validator.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class VmValidator < SimpleDelegator
```

Comment:
I am not sure what comment you want. 
They share a validator because there was code duplication and I would not want to write:
requests and request_templates share a validator because they both have to check the same vm values
because everytime someone uses this validator he would have to change the comment

Reply:
Fair enough. Thanks for the reply.